app/src/main/java/emu/skyline/EmulationActivity.kt

```diff
@@ -84,8 +84,8 @@ class EmulationActivity : AppCompatActivity(), SurfaceHolder.Callback, View.OnTo
     private var isEmulatorPaused = false
 
     private lateinit var pictureInPictureParamsBuilder : PictureInPictureParams.Builder
-    private val pauseIntentAction = "$packageName.EMULATOR_PAUSE"
-    private val muteIntentAction = "$packageName.EMULATOR_MUTE"
+    private val pauseIntentAction = "emu.skyline.EMULATOR_PAUSE"
```

Comment:
i don't think this will work for the dev* variants, maybe you could just latteinit this?

Reply:
A string constant? It's like a request code. Using the package name is just common practice.

Reply:
It is a little misleading, so I updated the names to better match the system ones.